assignment/assignment-8/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignment-8</title>
+    </head>
+    <body>
+        <h1>Contact Support</h1>
+        <form>
+            <label for="ticket"> Ticket </label>
+            <input
+                type="text"
+                id="name"
+                name="name"
+                required
+                minlength="4"
+                maxlength="8"
+                size="10"
+            />
+            <label for="email">Enter your email:</label>
+            <input type="email" id="email" pattern="" size="30" required />
+
+            <legend>Select a Issue:</legend>
+
+            <div>
+                <input
+                    type="radio"
+                    id="technical issue"
+                    name="issue"
+                    value="technical issue"
```

Comment:
Good, value provided. And correct. Could be wrong if you did like technicalIssue as It is not readable. id should technicalIssueRadio

Reply:
Technical Issue better one